src/Sylius/Bundle/ApiBundle/Command/Cart/BlameCart.php

```diff
@@ -16,23 +16,7 @@
 /** @experimental */
 class BlameCart
 {
-    /**
-     * @psalm-immutable
-     *
-     * @var string
-     */
-    public $shopUserEmail;
-
-    /**
-     * @psalm-immutable
-     *
-     * @var string
-     */
-    public $orderTokenValue;
-
-    public function __construct(string $shopUserEmail, string $orderTokenValue)
+    public function __construct(public string $shopUserEmail, public string $orderTokenValue)
```

Comment:
Now, both `shopUserEmail` and `orderTokenValue` are mutable. I'm wondering if maybe we should make a usage of `@experimental` and mark both fields a private and create `get*` like methods for both of them.

wdyt @Sylius/core-team @NoResponseMate @TheMilek

Reply:
We intend to develop this topic in the next task